mkdocs.yml

```diff
@@ -6,6 +6,15 @@ site_url: https://docs.percona.com/percona-monitoring-and-management/
 plugins:
   section-index: {}
 
+features:
```

Comment:
I believe this changes belongs in mkdocs-base.yml, since it is extended by multiple yml files.